Sources/OpenAPI/.create-api.yml

```diff
@@ -1,46 +1,21 @@
-# Modifier for all generated declarations
-access: public
-# Add @available(*, deprecated) attributed
-isAddingDeprecations: true
-# Available values: ["spaces", "tabs"]
+generate: [entities, paths, enums]
+module: AppStoreConnect_Swift_SDK
 indentation: tabs
-# Parses dates (e.g. "2021-09-29") using `NaiveDate` (https://github.com/CreateAPI/NaiveDate)
-isNaiveDateEnabled: false
-# Overrides file header comment
-fileHeaderComment: "// Generated by Create API\n// https://github.com/CreateAPI/CreateAPI"
-
+useNaiveDate: false
+fileHeaderComment: |
+  // Generated by Create API
+  // https://github.com/CreateAPI/CreateAPI
+  //
+  // swiftlint:disable all
 paths:
-  style: rest
-  # Generate response headers using https://github.com/CreateAPI/HTTPHeaders
-  isGeneratingResponseHeaders: true
-  # Add operation id to each request
-  isAddingOperationIds: false
-  # The types to import, by default uses "Get" (https://github.com/CreateAPI/Get)
-  imports: []
-  # Inline simple requests, like the ones with a single parameter
-  isInliningSimpleRequests: true
-  # Inline simple parametesr with few arguments.
-  isInliningSimpleQueryParameters: true
-  # Threshold from which to start inlining query parameters
-  simpleQueryParametersThreshold: 2
-  # Tries to remove redundant paths
-  isRemovingRedundantPaths: true
-  namespace: "APIEndpoint"
-
+  imports: [] # Prevent importing Get since we define our own Request type
+  namespace: APIEndpoint
+  overriddenBodyTypes:
+    application/a-gzip: Data
+    application/vnd.apple.xcode-metrics+json: Data
+    application/vnd.apple.diagnostic-logs+json: Data
 entities:
-  isGeneratingCustomCodingKeys: false
-  isGeneratingIdentifiableConformance: true
-
-comments:
-  # Generate comments
-  isEnabled: true
-  # Generate titles
-  isAddingTitles: true
-  # Generate description
-  isAddingDescription: true
-  # Generate examples
-  isAddingExamples: true
-  # Add links to the external documenatation
-  isAddingExternalDocumentation: true
-  # Auto-capitalizes comments
-  isCapitalizationEnabled: true
+  includeIdentifiableConformance: true
+  defaultType: struct
+  typeOverrides:
+    MetricCategory: finalClass
```

Comment:
And this is to suppress the following warning that was introduced:

> WARNING: Entity 'MetricCategory' cannot be generated as a struct because it has a stored property that recursively contains itself. Explicitly define an override using 'entities.typeOverrides' or ignore the entity/property using 'entities.ignore' to fix this issue.